scripts/install.sh

```diff
@@ -528,6 +528,23 @@ elif [ "$format" = "tar" ]; then
       fi
     fi
 
+    # check for an existing Doppler binary
+    existing_install_dir=$(which doppler)
```

Comment:
`which` isn't available on some platforms, so you'll want to use `command -v` here